pyccel/parser/semantic.py

```diff
@@ -3498,16 +3491,24 @@ def _visit_ClassDef(self, expr):
         #      - wouldn't be better if it is done inside ClassDef?
 
         name = expr.name
+        # remove quotes for str representation
         name = name.replace("'", '')
+
+        parent = {s: self.scope.find(s, 'classes') for s in expr.superclasses}
+        if any(s is None for s in parent.values()):
```

Comment:
For consistency here I would use the variable name `c` instead of `s`

Reply:
Done a156ea82